docs/user-quick-start.md

```diff
@@ -167,7 +167,8 @@ enough resources:
 
    For more info see
    [Submariner subctl installation](https://submariner.io/operations/deployment/subctl/).
-   Version v0.18.0 or later is required. Tested with version v0.20.1.
+   Tested with version v0.22.0.
+   Version v0.21.2 or later is required.
```

Comment:
Version v0.22.0 is later than  v0.21.2. but it is broken, maybe change the message?